src/Features/Core/Portable/EmbeddedLanguages/RegularExpressions/RegexHelpers.cs

```diff
@@ -37,26 +37,26 @@ public static RegexTrivia CreateTrivia(RegexKind kind, VirtualCharSequence virtu
     /// 'a' this will map to actual '\a' char (the bell character).  However, for something like
     /// '(' this will just map to '(' as that's all that \( does in a regex.
     /// </summary>
-    public static VirtualChar MapEscapeChar(VirtualChar ch)
+    public static int MapEscapeChar(VirtualChar ch)
         => ch.Value switch
         {
-            'a' => VirtualChar.Create(new Rune('\u0007'), ch.Span),    // bell
-            'b' => VirtualChar.Create(new Rune('\b'), ch.Span),        // backspace
-            'e' => VirtualChar.Create(new Rune('\u001B'), ch.Span),    // escape
-            'f' => VirtualChar.Create(new Rune('\f'), ch.Span),        // form feed
-            'n' => VirtualChar.Create(new Rune('\n'), ch.Span),        // new line
-            'r' => VirtualChar.Create(new Rune('\r'), ch.Span),        // carriage return
-            't' => VirtualChar.Create(new Rune('\t'), ch.Span),        // tab
-            'v' => VirtualChar.Create(new Rune('\u000B'), ch.Span),    // vertical tab
```

Comment:
we weren't using the VirtualChar-ness in any real way in this helper.  all the callers just called .Value off of it.  So this just collapses to that.